test/org/sagebionetworks/bridge/dynamodb/DynamoHealthDataDaoDdbTest.java

```diff
@@ -86,4 +99,62 @@ public void ddbRoundtrip() throws Exception {
             mapper.delete(savedRecord);
         }
     }
+
+    @Test
+    public void getRecordsByHealthCodeCreatedOnSchemaId() throws IOException {
```

Comment:
We should also add tests for:
* different health code
* createdOn out of range
* different schema ID